packages/lesswrong/components/posts/RecombeePostsList.tsx

```diff
@@ -72,14 +72,17 @@ export const stickiedPostTerms: PostsViewTerms = {
   forum: true
 };
 
-export const RecombeePostsList = ({ algorithm, settings, limit = 15, classes }: {
+export const RecombeePostsList = ({ algorithm, settings, limit = 15, showRecommendationsIcon = false, classes }: {
```

Comment:
naming of `showRecommendationsIcon` doesn't match `showRecommendationIcon` elsewhere (has an extra `s`)